client/src/pages/calculator.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from "react";
```

Comment:
Getting cannot read properties of null (reading 'useRef') error when going to calculator page. Is there a new dependency?

Reply:
New dependency: react-bootstrap

Reply:
Same issue ^